.envs/.local/.django

```diff
@@ -14,16 +14,31 @@ CELERY_FLOWER_USER=kJtrEYByunMyFNPJeALMRoyxcRTInfFw
 CELERY_FLOWER_PASSWORD=v7SFcfYcMLeAp6xDnFmxom9rNClFeTgmZz1wNn0HGSMPN1tbjSkxbeIKklPe2H8a
 
 # Jira
-# JIRA_SERVER=https://jira.server
-# JIRA_USERNAME=username
-# JIRA_PASSWORD=password
+JIRA_SERVER=https://jira.server
+JIRA_USERNAME=
+JIRA_PASSWORD=
+JIRA_CELL_ROLE_ACCOUNT=
+JIRA_CELL_ROLES=
 
 # Google
-# GOOGLE_API_PRIVATE_KEY=private_key
-# GOOGLE_API_CLIENT_EMAIL=client_email
-# GOOGLE_SPILLOVER_SPREADSHEET=spreadsheet_id
+GOOGLE_API_PRIVATE_KEY=
+GOOGLE_API_CLIENT_EMAIL=
+GOOGLE_SPILLOVER_SPREADSHEET=
+GOOGLE_CONTACT_SPREADSHEET=
+GOOGLE_AVAILABILITY_RANGE=
+GOOGLE_ROTATIONS_SPREADSHEET=
+GOOGLE_ROTATIONS_RANGE=
 
 # JWT
-# DJANGO_JWT_EXPIRATION_MINUTES=5
-# DJANGO_JWT_ALLOW_REFRESH=True
-# DJANGO_JWT_REFRESH_EXPIRATION_DAYS=7
+# JWT_ACCESS_TOKEN_LIFETIME_HOURS=3
+# JWT_REFRESH_TOKEN_LIFETIME_DAYS=30
+
+CELERY_BROKER_URL=redis://redis:6379/0
+DATABASE_URL=postgres://WmBQECsnGEnJEoysAOQIgHDHpxnNIKYt:WkcsI7Uuu3Jq7T1qylr9D3LI3WDVYszitV7p4uznJ4Ke9j7lAcuonprZ885hj8pJ@postgres:5432/sprints
```

Comment:
@Agrendalath Should we replace username and password with placeholders here?

Reply:
Sure, this looks confusing. I renamed them.

Reply:
@Agrendalath Thanks!